xarray/tests/test_dataarray.py

```diff
@@ -1668,6 +1668,11 @@ def test_squeeze_drop(self):
         actual = array.squeeze(drop=False)
         assert_identical(expected, actual)
 
+        array = DataArray([[[0., 1.]]], dims=['dim_0', 'dim_1', 'dim_2'])
+        expected = DataArray([[0., 1.]], dims=['dim_1', 'dim_2'])
+        actual = array.squeeze(axis=0)
+        assert_identical(expected, actual)
+
```

Comment:
For the better coverage, can you add tests for tuple axis case?
Also, it would be nice if we could have tests to make sure our code certainly raises an Error if both dim and axis are given.